README.md

```diff
@@ -40,26 +40,44 @@ auspice --datasetDir auspice/
 
 Build output goes into the directories `data/`, `results/` and `auspice/`.
 
-
 ## Configuration
 
 Configuration takes place entirely with the `Snakefile`. This can be read top-to-bottom, each rule
 specifies its file inputs and output and also its parameters. There is little redirection and each
 rule should be able to be reasoned with on its own.
 
 
-### fauna / RethinkDB credentials
-
-This build starts by pulling sequences from our live [fauna][] database (a RethinkDB instance). This
-requires environment variables `RETHINK_HOST` and `RETHINK_AUTH_KEY` to be set.
-
-If you don't have access to our database, you can run the build using the
-example data provided in this repository.  Before running the build, copy the
-example sequences into the `data/` directory like so:
-
-    mkdir -p data/
-    cp example_data/zika.fasta data/
-
+## Input data
+
+This build starts by downloading sequences from
+[https://data.nextstrain.org/files/zika/sequences.fasta.xz](data.nextstrain.org/files/zika/sequences.fasta.xz)
+and metadata from
+[https://data.nextstrain.org/files/zika/metadata.tsv.gz](data.nextstrain.org/files/zika/metadata.tsv.gz).
```

Comment:
These links are broken because the URL-part doesn't include the scheme (`https://`). When the link text should be the same as the URL, I'd suggest relying on auto-linking of base URLs. 

```suggestion
https://data.nextstrain.org/files/zika/sequences.fasta.xz
and metadata from
https://data.nextstrain.org/files/zika/metadata.tsv.gz.
```